### ISSUE TITLE
Allow use of `sass` executable found in PATH for offline usage

### ISSUE THREAD

**Issue by akram** [cleydyr/dart-sass-maven-plugin#40]
It could be useful for offline mode and air-gapped developers to allow the plugin to use the `sass` executable to be found in `PATH` instead of downloading or maintaining a `<cachedFilesDirectory>`

Could this be implement with an additional parameter or as a default failsafe behaviour?

**Comment by cleydyr** [comment]
Hey, Akram. Certainly, we could add a new parameter in future versions. The idea of trying to fall back to searching the path for the sass executable sounds interesting to me as well.

I'd welcome a PR implementing the suggestions. :)

**Comment by MilosHavranek** [comment]
We migrated to libsass-maven-plugin as it seems to be maintained and doesn't require internet connection. Which leads to repeatable builds along deterministic results.

```pom.xml
        <plugin>
          <groupId>com.gitlab.haynes</groupId>
          <artifactId>libsass-maven-plugin</artifactId>
        </plugin>
```

**Comment by cleydyr** [comment]
Please be aware: libsass, which the libsass-maven-plugin wrapper uses under the hood, is [deprecated](https://sass-lang.com/libsass/). That was the main reason I developed this Maven plugin, by the way.

**Comment by cleydyr** [comment]
Additionally, this plugin doesn't require an Internet connection. We have [a dedicated section in the README](https://github.com/cleydyr/dart-sass-maven-plugin?tab=readme-ov-file#offline-operation) about offline operation.

**Comment by MilosHavranek** [comment]
I was thinking a good way would be to reference webjar as it can also be replaced in pom.xml as plugin dependency, but unfortunately the webjar doesnt have a up to date version of sass. And honestly its not a good way to require something outside the build system, even if its offline.

**Comment by MilosHavranek** [comment]
Good catch about libsass though.. I'd say if pre-1.0 behavior could be added back with the option to provide newer sass version like it is now, it could be just enough. I care more about reproducibility of the build than to have bleeding edge release of sass. Auto-downloading latest version certainly breaks the build reproducibility. It's the kind of thing that would be expected somewhere like in npm, but not in java development.

**Comment by cleydyr** [comment]
I get your points, and that may be a different discussion. This issue deals with having this plugin use the Dart Sass executable in the operating system's `PATH`. I understand that the `PATH` is outside the build system, too.

Automatically downloading the _latest_ version is the _default_ behavior, but it's in no way _required_. You can get reproducible builds by pinning the Dart Sass version in the plugin options. If, in addition, you prefer to operate offline for any reason, you can also do so. Both cases are covered in the README.

As a last resort, you can use pre-1.0 versions of the plugin from the repository.

**Comment by MilosHavranek** [comment]
Thx, i figured that eventually. Also unfortunately webjar lacks the native executables. One way to solve this can be to upload the binaries to private maven repository and have it unpacked before the plugin runs into the cache folder. Which in the end might just be enough. I may post some example here later.
